packages/react/src/ReactContext.js

```diff
@@ -30,6 +30,10 @@ export function createContext<T>(defaultValue: T): ReactContext<T> {
     // These are circular
     Provider: (null: any),
     Consumer: (null: any),
+
+    // Add these to use same hidden class in VM as ServerContext
+    _defaultValue: (undefined: any),
+    _globalName: (undefined: any),
```

Comment:
Sometimes undefined is actually treated as a missing property. Might want to use `null` as a placeholder just to avoid that risk.